### DISCUSSION TITLE
How to use this app in a project.

### DISCUSSION THREAD

**Discussion by rakhiygit** [shacker/django-todo#137] [Q&A]
I am new to Django and have created the polls app from django documentation. You might have guessed i am below beginner stage. So, If you guys can point me to an article or wiki or any documentation to run this todo app in a project. It would be great.

**Comment by shacker** [comment]
@rakhiygit First you need to get a Django site up and running. Could be one that you made, or it could be the sample site provided with this project, [GTD](https://github.com/shacker/gtd).

Once you have a site running, you're going to install this reusable app into the project using the  instructions in the readme. 

Hope this helps,
Scot
